app/controllers/api/v1/joke_getter_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::JokeGetterController < ApplicationController
```

Comment:
If you want to follow convention, use `Api::V1::JokesController`

Reply:
Done